File: widgetssdk/src/main/java/com/glia/widgets/chat/domain/UpdateFromCallScreenUseCase.java
package com.glia.widgets.chat.domain;

import com.glia.widgets.chat.data.ChatScreenRepository;

public class UpdateFromCallScreenUseCase {
    private final ChatScreenRepository chatScreenRepository;

    public UpdateFromCallScreenUseCase(ChatScreenRepository chatScreenRepository) {
        this.chatScreenRepository = chatScreenRepository;
    }

    public void updateFromCallScreen(boolean fromCallScreen) {
        chatScreenRepository.setFromCallScreen(fromCallScreen);
    }
}

File: widgetssdk/src/main/java/com/glia/widgets/chat/domain/UpdateFromCallScreenUseCase.kt
package com.glia.widgets.chat.domain

import com.glia.widgets.chat.data.ChatScreenRepository

internal class UpdateFromCallScreenUseCase(private val chatScreenRepository: ChatScreenRepository) {
    operator fun invoke(fromCallScreen: Boolean) {
        chatScreenRepository.setFromCallScreen(fromCallScreen)
    }
}
